### PR TITLE
add overworld user and developer readme

### PR DESCRIPTION
**Pull request by cdingm**
Closes Gamify-IT/issues#329

### CODE REVIEW

File: user-manuals/overworld/README.md [hunk 1]
```diff
@@ -5,3 +5,27 @@ It is the main interface for students to interact with the platform.
 
 The overworld consists of multiple regions and dungeons.
 Throughout the world, students can discover course contents in form of [minigames](../README.md) and NPC dialogues.
+![overworld](../../images/overworld.webp)
+
+- On the screenshot you see the player in the middle.
+- He can walk around and interact with various objects, such as the NPC on the left.
+- There is also a HUD which displays a minimap and other useful information
```
Comment (delvh) [inline user-manuals/overworld/README.md:12]:
```suggestion
- The player is in the center of the screenshot
- They can walk around and interact with various objects such as the NPC on the left.
- There is a HUD which displays a minimap, and other useful information
```

File: user-manuals/overworld/README.md [hunk 2]
```diff
@@ -5,3 +5,27 @@ It is the main interface for students to interact with the platform.
...
+
+![npc dialogue](../../images/npc-dialogue.webp)
+
+- The Lecturer can configure what the NPCs talk
+- they could give tips about a topic for example
```
Comment (delvh) [inline user-manuals/overworld/README.md:17]:
```suggestion
- the lecturer can configure what the NPCs say
- they can use that to give tips about a topic for example
```

**Comment by delvh** [comment]
I'd approve if this PR wasn't WIP

**Comment by cdingm** [comment]
> I'd approve if this PR wasn't WIP

yeah i wasnt sure if it was sufficient

**Review by delvh — APPROVED** [review]

**Pull request merged**